datasets/index.md

```diff
@@ -7,12 +7,17 @@ has_children: true
 
 # Data Sources
 
-Data in the Data Commons Graph comes from a variety of sources, each of which often includes multiple surveys. Some sources/surveys include a very large number of variables, some of which might not yet have been imported into Data Commons. The sources have been grouped by category and are listed alphabetically within each category.
+Data in the Data Commons Graph comes from a variety of sources, each of which often includes multiple surveys. Some sources/surveys include a very large number of variables, some of which might not yet have been imported into Data Commons. The sources have been grouped by vertical and are listed alphabetically within each vertical.
```

Comment:
nit: Wonder if "category" is the more user-friendly term?

Reply:
done